### DISCUSSION TITLE
Publish to docker hub

### DISCUSSION THREAD

**Discussion by devdems** [postalserver/postal#1819] [Feature suggestions]
Hi,

is there a reason why there is no official docker published in https://hub.docker.com?

**Comment by willpower232** [comment]
Deploying from github is easier to githubs own registry I believe and doesn't docker take down images which aren't pulled after a certain amount of time if you aren't paying?

**Comment by EsdrasCaleb** [comment]
No you can deploy easy to dockerhub with the same CI I have one in my repo:
https://github.com/EsdrasCaleb/moodle-docker-php-production/blob/main/.github/workflows/docker-publish.yml

I am also tring to deploy postal as a one click app off caprover and can only do this if I have an official image in dockerhub or in anyplace
